packages/server/src/appMigrations/migrations/20240604153647_initial_sqs.ts

```diff
@@ -0,0 +1,59 @@
+import { context } from "@budibase/backend-core"
+import { allLinkDocs } from "../../db/utils"
+import LinkDocumentImpl from "../../db/linkedRows/LinkDocument"
+import sdk from "../../sdk"
+import env from "../../environment"
+import { DBError } from "@budibase/types"
+
+const migration = async () => {
+  const linkDocs = await allLinkDocs()
```

Comment:
Does this have the potential to be very large? Do we instead want to paginate?

Reply:
I think the scale of this is likely to be small enough (its very hard to create a lot of internal relationships today) that pagination adds enough complexity that its not worth it.